docs/modalities.md

```diff
@@ -277,6 +278,18 @@ provides two options:
 This modality is an optional installation parameter and will default to the `Burnable` mode if not provided. However, this
 mode cannot be changed once the contract has been installed. The mode is set by passing a `u8` value to the `burn_mode` runtime argument.
 
+## OperatorBurnMode
+
+The `OperatorBurnMode` modality dictates whether tokens minted by a given instance of an NFT contract can be burnt by an operator (thus accounts, contracts or contracts from a package approved for all by an owner). This modality
```

Comment:
```suggestion
The `OperatorBurnMode` modality dictates whether tokens minted by a given instance of an NFT contract can be burned by an operator (thus accounts, contracts or contracts from a package approved for all by an owner). This modality provides two options:
```

Reply:
well I count...ed 37 burnt doc and code together and 8 burned. Google says both are acceptable since 400 years https://www.grammarly.com/blog/burnt-burned/ but apprently it is still debat...ed.